features/dd-sdk-android-trace/src/test/kotlin/com/datadog/trace/logger/DatadogCoreTracerLoggerTest.kt

```diff
@@ -0,0 +1,335 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.trace.logger
+
+import com.datadog.android.api.InternalLogger
+import com.datadog.android.trace.utils.verifyLog
+import com.datadog.android.utils.forge.Configurator
+import fr.xgouchet.elmyr.Forge
+import fr.xgouchet.elmyr.annotation.StringForgery
+import fr.xgouchet.elmyr.annotation.StringForgeryType
+import fr.xgouchet.elmyr.junit5.ForgeConfiguration
+import fr.xgouchet.elmyr.junit5.ForgeExtension
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.extension.ExtendWith
+import org.junit.jupiter.api.extension.Extensions
+import org.mockito.Mock
+import org.mockito.junit.jupiter.MockitoExtension
+import org.mockito.junit.jupiter.MockitoSettings
+import org.mockito.quality.Strictness
+import java.util.Locale
+
+@Extensions(
+    ExtendWith(MockitoExtension::class),
+    ExtendWith(ForgeExtension::class)
+)
+@MockitoSettings(strictness = Strictness.LENIENT)
+@ForgeConfiguration(value = Configurator::class)
+internal class DatadogCoreTracerLoggerTest {
+
+    private lateinit var testedLogger: DatadogCoreTracerLogger
+
+    @StringForgery
+    lateinit var fakeMessage: String
+
+    private lateinit var expectedFakeMessage: String
```

Comment:
minor suggestion: fakes lingvo is usually just for the setup phase. And also for the consistency, because in some assertions it is already called as `expectedMessage`.

Another suggestion is to remove it from the properties as have as a function.

```suggestion
    private lateinit var expectedMessage: String
```